cli/src/diff_util.rs

```diff
@@ -458,7 +494,11 @@ pub fn show_color_words_diff(
                 let left_content = diff_content(&path, left_value)?;
                 let right_content = diff_content(&path, right_value)?;
                 writeln!(formatter.labeled("header"), "{description} {ui_path}:")?;
-                show_color_words_diff_hunks(&left_content, &right_content, formatter)?;
+                if left_content.is_binary || right_content.is_binary {
+                    writeln!(formatter.labeled("empty"), "    (binary)")?;
```

Comment:
i got lazy and didn't add a new label; i can add a new `binary` label if you prefer

Reply:
I think a new binary label would be better. It should only take adding a line to [`cli/src/config/colors.toml`](https://github.com/martinvonz/jj/blob/01825953aca5b9aab6d7e9db7777818482f99914/cli/src/config/colors.toml#L4), I believe.